extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -2801,8 +2801,8 @@ public void close()
         EasyMock.createNiceMock(DruidNode.class),
         new LookupNodeService("tier"),
         new DataNodeService("tier", 1, ServerType.INDEXER_EXECUTOR, 0),
-        new TaskReportFileWriter(reportsFile)
-    );
+        new TaskReportFileWriter(reportsFile),
+        false);
```

Comment:
A way to avoid adding this boolean in multiple tests is to have 1 ctor which defaults to false, and a new ctor which gets the boolean

Reply:
Not sure Guice can inject into multiple constructors but I'll look it up